src/chapters/angular-wk1/studio.rst

```diff
@@ -0,0 +1,61 @@
+Studio: Angular, Part 1
```

Comment:
I like the goal of this studio, and I think you provide a good look forward by referencing how it will be improved in the following classes.

Two additions that would be helpful:
1) A 'final product' screen shot or live site to let the students see the daily goal.
2) Link to starter code that already has ``hello.component.ts`` removed, or since the exercises have a 3 column format, maybe encourage the students to modify their existing work.